class.jetpack-idc.php

```diff
@@ -139,10 +168,17 @@ function enqueue_idc_notice_files() {
 			)
 		);
 
+		wp_register_style(
+			'jetpack-dops-style',
+			plugins_url( '_inc/build/admin.dops-style.css', JETPACK__PLUGIN_FILE ),
+			array(),
+			JETPACK__VERSION
+		);
+
```

Comment:
I don't think we should be enqueuing the entirety of the dops styles.  It's a lot of css.  @jeffgolenski would you rather us enqueue the entire stylesheet or abstract/duplicate the styles we need?